Tests/FluentAssertions.Specs/Collections/CollectionAssertionSpecs.ContainInOrder.cs

```diff
@@ -100,6 +100,20 @@ public void When_passing_in_null_while_checking_for_ordered_containment_it_shoul
                 "Cannot verify ordered containment against a <null> collection.*");
         }
 
+        [Fact]
+        public void Collections_contain_the_empty_sequence()
+        {
+            // Assert
+            new[] { 1 }.Should().ContainInOrder(new int[0]);
+        }
+
+        [Fact]
+        public void Collections_do_not_not_contain_the_empty_sequence()
```

Comment:
I struggled finding a satisfying name for this test, so all ears if you can come up with a more fitting name.

Reply:
`Every_collection_contains_an_empty_collection`?